apps/platform/utils/helpers.ts

```diff
@@ -129,3 +129,16 @@ export const getNavigationTabs = (currentUserRole, projectUrl) => {
 
   return defaultTabs;
 };
+
+export const download = (filename, text) => {
```

Comment:
Great work on the function, but I'd recommend renaming `download` to something more descriptive like `downloadAsTextFile`. This will make the function's purpose more clear in this file and avoid potential naming conflicts. What do you think?

Reply:
Sure @ttebify, Thanks